Mathlib/CategoryTheory/Limits/Lattice.lean

```diff
@@ -188,11 +182,11 @@ def colimitCocone (F : J ⥤ α) : ColimitCocone F where
 -- It would be nice to only use the `Inf` half of the complete lattice, but
 -- this seems not to have been described separately.
```

Comment:
I don't know if you want to change this here, but I don't think this comment applies: this is described separately now, but more importantly a complete lattice in terms of only Inf is identical to a complete lattice, so I don't think there's any actionable point in this comment.

Reply:
Yes, it is a better design to always use `CompleteLattice`. I have removed this comment.